src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.scss

```diff
@@ -21,22 +18,22 @@ $content-width: $gene_image_width;
 }
 
 .proteinSummaryTop {
-  align-items: center;
   display: grid;
-  grid-template-columns: [references] 300px [download] 395px;
+  grid-template-columns: [references] 350px [download] 345px;
 }
 
 .downloadWrapper {
   display: flex;
   justify-content: flex-end;
   grid-column: download;
+  align-items: flex-start;
```

Comment:
It seems that the reason you are adding this flex-start is because the height of the `.downloadWrapper` becomes too large:

![image](https://user-images.githubusercontent.com/6834224/116384872-2e05b080-a810-11eb-9459-47fd8a48ea31.png)

and the button gets distorted without this rule:

<img src="https://user-images.githubusercontent.com/6834224/116384945-3f4ebd00-a810-11eb-9719-4318af4be4c7.png" width="400px"/>

I would argue that what we really should try to do is to prevent the `.downloadWrapper` from getting this high in the first place. The reason it is too high is because it is trying to occupy the full height of the grid cell. If you add `align-items: start` to the `.proteinSummaryTop` class, the `.downloadWrapper` element will keep its natural height:

![image](https://user-images.githubusercontent.com/6834224/116385622-e0d60e80-a810-11eb-8a37-19233eb685b2.png)

(The same result can be achieved by adding `align-self: start` to `.downloadWrapper`; but `align-items: start` on `.proteinSummaryTop`) makes more sense.